### ISSUE TITLE
Block expansions in org-mode not sourced from the user folder

### ISSUE THREAD

**Issue by shrysr** [jkitchin/scimax#207]
Earlier -  I was able to use the snippet below to enable the expansion desired. This was placed in my org configuration in the user folder. However, this is no longer working after merging with the latest scimax master.  Scimax-ivy-yas is cool! However, it does not seem to include the /user/snippets folder. 

```
(add-to-list 'org-structure-template-alist
	     '("ip" "#+BEGIN_SRC ipython :session :results output org drawer\n?\n#+END_SRC"
	       "<src lang=\"ipython\">\n?\n</src>"))
```

**Comment by jkitchin** [comment]
The org snippet you show should not be necessary in scimax anymore, all those headers are defaults now. so just 

```
#+BEGIN_SRC ipython

#+END_SRC
```
is ok.

I am surprised to hear scimax-ivy-yas  doesn't include your user templates, it does for me. Maybe you need to try reloading the snippets? M-x yas-reload-all, and check that the user directory is listed in yas/root-directory.

**Comment by shrysr** [comment]
Thanks! I did not realise that the headers were on by default. That makes it convenient indeed. Though I've been happily using scimax awhile - I have just started exploring the details.

Sorry : I just rechecked and found that I was confusing the org snippets with yasnippet. scimax-ivy-yas does list the snippets in the user directory. It is actually very convenient.  I will go ahead and close this issue.
